packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -4707,6 +4707,23 @@ describe('StripeHelper', () => {
         });
       });
 
+      it('extracts expected details from an invoice of an upgrade', async () => {
+        const fixture = deepCopy(invoicePaidSubscriptionCreate);
+        fixture.lines.data.push(fixture.lines.data[0]);
+        fixture.lines.data[0].type = 'invoiceitem';
+        fixture.lines.data[1].period.end = 1593038132;
+        const result = await stripeHelper.extractInvoiceDetailsForEmail(
+          fixture
+        );
+        assert.isTrue(stripeHelper.allAbbrevProducts.called);
+        assert.isFalse(mockStripe.products.retrieve.called);
+        sinon.assert.calledTwice(expandMock);
+        assert.deepEqual(result, {
+          ...expected,
+          nextInvoiceDate: new Date('2020-06-24T22:35:32.000Z'),
```

Comment:
It's hard for me to tell that this Date is the same as `1593038132` above. Can we use the same method as used in `extractInvoiceDetailsForEmail` -- namely `nextInvoiceDate: new Date(1593038132 * 1000)`?

Reply:
Made the recommended updates and added some other fixes in the test.